src/main/java/equipments/EquipmentStatus.java

```diff
@@ -0,0 +1,26 @@
+package equipments;
+
+import java.util.HashMap;
+import java.util.List;
+
+public class EquipmentStatus {
+    public static <T extends Equipment> String teamMemberEquipmentDetails(HashMap<String, List<T>> memberEquipment) {
```

Comment:
avoid using `static`- it makes it harder to change implementations later down the line and is impossible to unit-test. there is nothing wrong with creating an instance of a functional class (it's better for the garbage collector too)

Reply:
also, why use `T extends Equipment` instead of just `Equipment`?